patches/ng-packagr+11.2.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/ng-packagr/lib/ng-package/entry-point/write-bundles.transform.js b/node_modules/ng-packagr/lib/ng-package/entry-point/write-bundles.transform.js
+index ecb19da..2cd9df4 100644
+--- a/node_modules/ng-packagr/lib/ng-package/entry-point/write-bundles.transform.js
++++ b/node_modules/ng-packagr/lib/ng-package/entry-point/write-bundles.transform.js
+@@ -54,7 +54,7 @@ exports.writeBundlesTransform = transform_1.transformFromPromise(async (graph) =
+             format: 'umd',
+             dest: umd,
+             cache: cache.rollupUMDCache,
+-            transform: downlevel_plugin_1.downlevelCodeWithTsc,
++            //transform: downlevel_plugin_1.downlevelCodeWithTsc,
```

Comment:
This is a last resort. I'm not opposed to it if we can't solve this another way, but would prefer exploring other options first. See https://github.com/single-spa/single-spa-angular/issues/370#issuecomment-875290631 where i discuss an alternative